packages/eslint-config-audius/src/index.js

```diff
@@ -4,13 +4,11 @@ module.exports = {
     es6: true,
   },
   extends: [
-    "standard",
```

Comment:
Hmmm, why aren't these needed anymore?

Reply:
yeah good catch.... will re-run with it added, that was meant to be there!